components/channel_info_rhs/components/linelimiter.tsx

```diff
@@ -53,7 +53,7 @@ const LineLimiterBase = ({children, maxLines, lineHeight, moreText, lessText, er
             <>
                 <div
                     className={className}
-                    css={{maxHeight}}
```

Comment:
looks like a completely unrelated change sneaked there, but looks good to me anyway.

Reply:
yes, i don't know why we did caught that before in the pr